homeassistant/components/no_ip.py

```diff
@@ -0,0 +1,113 @@
+"""
+Integrate with NO-IP Dynamic DNS service.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/no_ip/
+"""
+import asyncio
+import base64
+import logging
+from datetime import timedelta
+
+import aiohttp
+import async_timeout
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    CONF_DOMAIN, CONF_PASSWORD, CONF_TIMEOUT, CONF_USERNAME, HTTP_HEADER_AUTH,
+    HTTP_HEADER_USER_AGENT, PROJECT_EMAIL)
+from homeassistant.helpers.aiohttp_client import SERVER_SOFTWARE
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'no_ip'
+
+INTERVAL = timedelta(minutes=5)
+
+DEFAULT_TIMEOUT = 10
+
+NO_IP_ERRORS = {
+    'nohost': "Hostname supplied does not exist under specified account",
+    'badauth': "Invalid username password combination",
+    'badagent': "Client disabled",
+    '!donator':
+        "An update request was sent with a feature that is not available",
+    'abuse': "Username is blocked due to abuse",
+    '911': "A fatal error on NO-IP's side such as a database outage",
+}
+
+UPDATE_URL = 'https://dynupdate.no-ip.com/nic/update'
+USER_AGENT = "{} {}".format(SERVER_SOFTWARE, PROJECT_EMAIL)
```

Comment:
That is already default

Reply:
No, the default is only `SERVER_SOFTWARE`. NO-IP wants [additionally](https://www.noip.com/integrate/request) an email address to be added the user agent.

Reply:
Can you update the URL to https://dynupdate.noip.com/nic/update ?  This is the URL that we are migrating to.

Reply:
Thanks @kbussche, URL updated.